packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -40,8 +40,10 @@ function createCompatSuite(
                         config.containerRuntime,
                         config.dataRuntime,
                     );
+                    // eslint-disable-next-line @typescript-eslint/no-invalid-this
                     Object.defineProperty(this, "__fluidTestProvider", { get: () => provider });
                 });
+                // eslint-disable-next-line @typescript-eslint/no-invalid-this
                 tests.bind(this)((reset: boolean = true) => {
```

Comment:
@microsoft/fluid-cr-test Do you know whether these are bugs?

Reply:
I have not fully understood how `this` works in all cases yet.  But I know Mocha sets `this` to some context object in some cases/ways, and it appears in the parameter list (but I don't think callers pass it - this is the part I don't totally get). so I think someone could explain why this is correct.  If this were not correct none (ish) of our tests would pass.